### DISCUSSION TITLE
Move channel to another Team

### DISCUSSION THREAD

**Discussion by perhaland** [pnp/powershell#219] [Ideas]
In some cases like reorganization there is a need to move a channel from one Team to another.
Could this be done as a pnp-powershell command?

**Comment by YuriySamorodov** [comment]
As of now now this is not supported at all as far as I know, but it is in Microsoft backlog. 
You may want to vot for this feature here:
https://microsoftteams.uservoice.com/forums/555103-public/suggestions/16939708-move-channels-into-other-teams
